api/relations/v0/lookup.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package kessel.relations.v0;
+
+import "google/api/annotations.proto";
+import "relations/v0/common.proto";
+
+option go_package = "ciam-rebac/api/relations/v0;v0";
+option java_multiple_files = true;
+option java_package = "org.project_kessel.api.relations.v0";
+
+service KesselLookupService {
+    rpc LookupSubjects(LookupSubjectsRequest) returns (stream LookupSubjectsResponse) {
+        option (google.api.http) = {
+            get: "/v1/subjects"
+        };
+    };
+}
+
+message LookupSubjectsRequest {
+    ObjectType subject_type = 1;
+    string subject_relation = 2;
+    string relation = 3;
+    ObjectReference resource = 4;
+    optional uint32 limit = 5;
+    optional string continuation_token = 6;
+}
+
+message LookupSubjectsResponse {
+    SubjectReference subject = 1;
+    string continuation_token = 2;
```

Comment:
We may want to adjust this schema to allow for conditional relations as well as exclusions from within a wildcard. Both of which may end up being relevant to notifications.

This is another thing we may want to follow up on (i.e. get this out the door at least for experimentation, and then follow up with an additional PR)

Reply:
Agree on follow-up